src/state/api.js

```diff
@@ -1,28 +0,0 @@
-import fetch from "node-fetch";
```

Comment:
Maybe we should bring this back in some form and use them in the components, if possible.